src/modules/admin/components/forms/FormStatusText.tsx

```diff
@@ -0,0 +1,41 @@
+import { Typography } from '@mui/material';
+import { FormIdentifierDetailsFragment, FormStatus } from '@/types/gqlTypes';
+
+interface Props {
+  identifier: FormIdentifierDetailsFragment;
+}
+const FormStatusText: React.FC<Props> = ({ identifier }) => {
```

Comment:
Just pulled this out from `src/modules/admin/components/forms/FormDefinitionDetailPage.tsx`, no changes